Dockerfile

```diff
@@ -1,13 +1,32 @@
-FROM python:3.9
+FROM python:3.9-slim
```

Comment:
I just change that on main to tiangolo/uvicorn-gunicorn:python3.9-slim. I'm using the prestart.sh from this